deploy/template.yaml

```diff
@@ -123,6 +123,26 @@ Resources:
           - ServerSideEncryptionByDefault:
               SSEAlgorithm: AES256
 
+  AccessLogsBucketPolicy:
+    Type: AWS::S3::BucketPolicy
+    Properties:
+      Bucket: !Ref AccessLogsBucket
```

Comment:
There is already a Bucket Policy on this bucket a few lines down. If you try adding this second one they'll compete with each other (as I've previously found out). You should addd this as an additional statement to the existing policy.